src/lib/isMobilePhone.js

```diff
@@ -74,6 +74,7 @@ const phones = {
   'fi-FI': /^(\+?358|0)\s?(4(0|1|2|4|5|6)?|50)\s?(\d\s?){4,8}\d$/,
   'fj-FJ': /^(\+?679)?\s?\d{3}\s?\d{4}$/,
   'fo-FO': /^(\+?298)?\s?\d{2}\s?\d{2}\s?\d{2}$/,
+  'fr-CI': /^(\+?255)\d{8}$/,
```

Comment:
Kindly check the [official ITU numbering plan](https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000310006PDFE.pdf) (page 4 and 5)
There they actually list all of the valid mobile phone number ranges.
Could you kindly revise the regexp accordingly please and also update the tests accordingly as well?

thank you!